prime-router/src/main/kotlin/tokens/AuthenticatedClaims.kt

```diff
@@ -1,14 +1,22 @@
 package gov.cdc.prime.router.tokens
 
+import gov.cdc.prime.router.Sender
+import io.jsonwebtoken.Claims
+import io.jsonwebtoken.Jwts
+
 /**
  * This represents a set of claims coming from Okta that have been authenticated (but may or may not
  * have been authorized). For convenience, certain claims have been pulled out into more readable/usable forms,
  * using helper methods in the class, called during object construction.
  */
-class AuthenticatedClaims(
-    val jwtClaims: Map<String, Any>,
-) {
-    // These are all derived from the raw jwtClaims.
+class AuthenticatedClaims {
```

Comment:
Despite now plopping both Okta claims and Server2Server claims into the same `AuthenticatedClaims` object, there are still differences.  Another pass at refactoring could be done, but I think it should be done in the context of a design proposal for how to consolidate "Group memberships" (coming from Okta) with "claims" and "scopes" (coming from server2server auth).

There are standard ways and rules for representing `scopes`, so I suggest we come up with a plan that allows for Okta Groups to map elegantly onto server2server scopes.   This is all well beyond the scope of this ticket.